commands.js

```diff
@@ -1044,16 +1044,56 @@ exports.commands = {
 			move = Commands.randommove.call(this, "2,viable", self, dummyRoom);
 		}
 
+		// Code for generating a random name based on types generated
+		// Start by accessing the array from the namelist.js file in the data folder.
+		var foundName = false;
+		var t1Name = '';
+		var type1array = Tools.shuffle(Data.Namelist[type[0]]);
+
+		var type2array = [];
+		if (type[1] === undefined) { //Checking for single type case
+			type2array = Tools.shuffle(type1array);
+		} else {
+			type2array = Tools.shuffle(Data.Namelist[type[1]]);
+		}
+
+		// Grab our first word for the name
+		Tools.shuffle(Data.Namelist[type[1]]);
+		for (var i = 0; i < type1array.length; i++) {
+			if(type1array[i].charAt(0) === '+') {
```

Comment:
You fixed almost all of the styling issues. The only one I still see is not having spaces after the `if` keywords. Thank you for correcting that though.